README.rst

```diff
@@ -362,7 +362,42 @@ Install Python bindings
 
 .. installation-end-marker-do-not-remove
 
-----
+Cross-compiling DALI C++ API for aarch64 Linux (Docker)
+-----------------------------------------
+Build the aarch64 Linux Build Container 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: bash
+    docker build -t dali_builder:aarch64-linux -f Dockerfile.build.aarch64-linux .
+
+Compile
+^^^^^^^
+From the root of the DALI source tree
+.. code-block:: bash
```

Comment:
You need a newline before the `.. code-block:: bash`

Reply:
Done